.gitignore

```diff
@@ -18,3 +18,6 @@ dist-newstyle
 postgrest.hp
 postgrest.prof
 __pycache__
+*.tix
```

Comment:
This is optional now I think, but doesn't hurt to leave it in for further experimentation on coverage